gocryptfs build failed on PPA for Ubuntu environment (v1.2.0-rc.1)

### Version of Apptainer

v1.2.0-rc1

### Expected behavior

1.2.0-rc.1, gocryptfs successfully built on PPA for Ubuntu environment

### Actual behavior

1.2.0-rc.1, gocryptfs build failed on PPA for Ubuntu environment

```
+ cd debian/gocryptfs-2.3.2
+ PATH=/tmp/appdebgo/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games GOCACHE=/tmp/appdebgo/cache ./build-without-openssl.bash
build.bash: warning: could not determine gocryptfs version
go: github.com/hanwen/go-fuse/v2@v2.3.0: Get "https://proxy.golang.org/github.com/hanwen/go-fuse/v2/@v/v2.3.0.info": dial tcp: lookup proxy.golang.org: no such host
go: github.com/aperturerobotics/jacobsa-crypto@v1.0.0: Get "https://proxy.golang.org/github.com/aperturerobotics/jacobsa-crypto/@v/v1.0.0.mod": dial tcp: lookup proxy.golang.org: no such host
go: github.com/aperturerobotics/jacobsa-crypto@v1.0.0: Get "https://proxy.golang.org/github.com/aperturerobotics/jacobsa-crypto/@v/v1.0.0.mod": dial tcp: lookup proxy.golang.org: no such host
go: could not create module cache: mkdir /sbuild-nonexistent: permission denied
make[1]: *** [debian/rules:69: override_dh_auto_configure] Error 1
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
make: *** [debian/rules:64: build] Error 2
```

### Steps to reproduce this behavior

upload apptainer source changes file to PPA for Ubuntu

### What OS/distro are you running

Rocky Linux 9

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -157,6 +157,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Fix memory usage calculation during apptainer compilation on RaspberryPi.
 - Fix misleading error when an overlay is requested by the root user while the
   overlay kernel module is not loaded.
+- Fix gocryptfs build procedures for deb package.
 
 ## v1.1.9 - \[2023-06-07\]
 

